src/main/webapp/i18n/de/statistics.json

```diff
@@ -0,0 +1,37 @@
+{
+    "statistics": {
+        "amountOfStudents": "# an Studenten",
+        "span": {
+            "day": "Tag",
+            "week": "Woche",
+            "month": "Monat",
+            "year": "Jahr"
+        },
+        "title": "Benutzer Statistiken",
+        "days": " Tagen",
+        "loggedInUsers": "Anzahl an Anmeldungen unterschiedlicher Benutzer in den letzten ",
+        "activeUsers": "Anzahl an Anmeldungen aktiver Benutzer in den letzten ",
+        "totalSubmissions": "Anzahl an Abgaben in den letzten ",
+        "exercises": {
+            "released": "Anzahl an veröffentlichten Aufgaben in den letzten ",
+            "due": "Anzahl an Aufgaben mit einer Deadline in den letzten "
+        },
+        "conductedExams": "Anzahl an abgehaltenen Klausuen in den letzten ",
+        "examParticipation": " -> und Anzahl Studenten, die daran teilgenommen haben: ",
+        "examRegistrations": " -> Die Anzahl an Studenten mit einer Registrierung für die Klausur: ",
+        "activeTutors": "Anzahl der Tutoren, die eine Bewertung erstellt haben in den letzten ",
+        "results": "Anzahl erstellter Resultate in den letzten ",
+        "resultFeedbacks": "-> mit der absoluten Feedbackanzahl: ",
+        "userLogins": "Benutzer Anmeldungen",
+        "submissions": "Abgaben"
+    },
+    "weekdays": {
```

Comment:
get rid of this

Reply:
[Done](https://github.com/ls1intum/Artemis/pull/2471/commits/499bb9fd70d8f4710fe3744189d0493f61a8db7d)